features/environment.py

```diff
@@ -107,13 +110,18 @@ def before_all(context: Context) -> None:
     userdata = context.config.userdata
     context.reuse_container = userdata.get("reuse_container")
     context.config = UAClientBehaveConfig.from_environ()
+    context.pdb = userdata.get("pdb")
```

Comment:
Could you move it to the line below `context.reuse_container = `?  So, we have `userdata` config together

Reply:
Thank you @patriciadomin. Good suggestion. I have moved it up and force pushed changes